code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -282,7 +282,7 @@
 /obj/machinery/atmospherics/components/unary/cryo_cell/relaymove(mob/user)
 	if(message_cooldown <= world.time)
 		message_cooldown = world.time + 50
-		to_chat(user, "<span class='warning'>[src]'s door won't budge!</span>")
+		to_chat(user, "<span class='warning'>\The [src]'s door won't budge!</span>")
```

Comment:
None of the cryo cell changes do anything.
It looks like you cancelled out the text with \

Reply:
Are you sure? In-game it (currently) appears as "cryo cell's door won't budge!". Is there some other issue going on here?

Reply:
You have a cancellation on all of the "The"s in your changes, so the compiler is ignoring it entirely
I had checked in-game and it was still "cryo cell's door won't budge!"